src/components/Table/Table.vue

```diff
@@ -76,7 +76,7 @@
                     :index="index"
                 >
                     <tr
-                        :class="tbodyTrClass"
+                        :class="typeof tbodyTrClass === 'function' ? tbodyTrClass(item) : tbodyTrClass"
```

Comment:
From what I remember, using methods in `template` is a big no-no. Since "[will be called every time the component updates](https://vuejs.org/guide/essentials/template-syntax.html#calling-functions)".

Reply:
It has been decided that his will be the developers choice.